recipes/hsm/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+required_conan_version = ">=1.32.1"
+
+class HsmConan(ConanFile):
+    name = "hsm"
```

Comment:
```suggestion
    name = "erikzenker-hsm"
```

I suggest renaming your package with author due name collision:

https://github.com/amaiorano/hsm
https://github.com/Franklin-Qi/HSM
https://github.com/howard-chan/HSM

Reply:
@erikzenker 
Can you please apply this + rename the folder?